### PR TITLE
Added unit tests, IT for package without version

### LINKED ISSUE #83: Support for uploading file without "version"
Example: https://github.com/aws/aws-sdk-php/blob/master/composer.json
This json file are lack of "version" key. When handling this minimal-package.json, the version() method in JsonPackage.class will throw error.
The version message maybe input by user when uploading. 
Composer.json is usually include in zip file, this is associate with this issue: https://github.com/artipie/composer-adapter/issues/23

### PR DESCRIPTION
**Pull request by genryxy**
Closes #83
Added unit tests, IT for package without version.
Also, extracted repeated method for creating empty ZIP.

### CODE REVIEW

File: src/test/java/com/artipie/composer/AstoRepositoryAddJsonTest.java [hunk 1]
```diff
@@ -144,8 +152,41 @@ void shouldDeleteSourceAfterAdding() throws Exception {
...
+        );
+        MatcherAssert.assertThat(
+            "Added package contains `version` entry",
+            pkgs.getJsonObject(vers.get()).getString("version"),
+            new IsEqual<>(vers.get())
+        );
+    }
+
+    @Test
+    void shouldFailToAddPackageWithoutVersionWithEmptyPassed() {
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/composer/AstoRepositoryAddJsonTest.java:178]: @genryxy test method name seems odd, please, check it

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy just one comment, please, take a look

**Comment by genryxy** [comment]
@olenagerasimova thanks, omitted the last part `WithEmptyPassed` in name of method

**Review by olenagerasimova — APPROVED** [review]
@genryxy thanks

**Pull request merged**